### PR TITLE
add an issue template

### PR DESCRIPTION
**Pull request by tych0**
There have always been a lot of questions-via-issues, but recently there
have also been a lot of issues where the first round trip is "please post
your config" or "please elaborate". Let's try to head that off with an
issue template.

Signed-off-by: Tycho Andersen <tycho@tycho.ws>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Please do not ask general questions here! There are [comunnity
+contact](https://github.com/qtile/qtile#community) options for that.
```
Comment (ramnes) [inline .github/ISSUE_TEMPLATE.md]: I'm +1 for the template but I really don't mind the questions here. I actually prefer here than on the mailing list, for example.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Issue description
+
+A brief discussion of what failed and how it failed. A description of what you
+tried is helpful, i.e. "When I use lazy.kill() on a window nothing happens"
```
Comment (ramnes) [inline .github/ISSUE_TEMPLATE.md]: Nothing happens is pretty vague too, maybe talk about the logs?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+instead of "Closing windows doesn't work".
+
+# Qtile version
+
+Please include the exact commit hash of the version of Qtile that failed.
```
Comment (ramnes) [inline .github/ISSUE_TEMPLATE.md]: People installing through their distribution package manager won't know the commit hash for sure

**Comment by ramnes** [comment]
LGTM overall, way overdue

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Please do not ask general questions here! There are [comunnity
```
Comment (pawamoy) [inline .github/ISSUE_TEMPLATE.md]: typo: comunnity

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (pawamoy) [reply .github/ISSUE_TEMPLATE.md]: If properly tagged, questions on the bugtracker are great to feed the wiki :slightly_smiling_face:

Comment (tych0) [reply .github/ISSUE_TEMPLATE.md]: I can drop this, but then it seems like we should dismantle the mailing list all together if nobody wants to use it.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (tych0) [reply .github/ISSUE_TEMPLATE.md]: Then they need to find it out :). This seems like useful information for debugging. Now, one thing I do notice is that qtile --version doesn't print it, so we should fix that.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ramnes) [reply .github/ISSUE_TEMPLATE.md]: I like this idea, but then to each his favorites tools I guess, so I wouldn't mind keeping the current status to reach more people. I didn't find this unmanageable so far. The only thing I don't want is adding more tools than we have currently, e.g. dealing with another chat than IRC unless we all stop using IRC.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (ramnes) [reply .github/ISSUE_TEMPLATE.md]: Yes, let's fix that first and say to past the output here

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (tych0) [reply .github/ISSUE_TEMPLATE.md]: Ok. I find it annoying that we use issues for people to ask questions. So maybe we can all agree that we can close questions immediately, or something? I'd prefer for the mailing list to be about qustions, issues to be about issues, etc.

**Comment by tych0** [comment]
Merging this for now. If someone has time to look into how to always get the right git version number, that would be handy.

**Comment by ramnes** [comment]
Ah damn, my server got shutdown and thus I got disconnected from IRC, and then taken over by kids, so I'm not sure what the end of our discussion was! I still really think it's not welcoming / user friendly to tell people not to ask questions here. Do you like the idea of using the GitHub action actions/stale so that old issues or PR get closed automatically, and then remove the first paragraph of the template here?

**Comment by tych0** [comment]
Yeah, that sounds great!

**Pull request merged**